### PR TITLE
Introduce log.source.address and log.file.path for 7.x compatiblity

### PR DESCRIPTION
**Pull request by ruflin**
Related to https://github.com/elastic/beats/pull/8902 but adding the fields instead of replacing

### CODE REVIEW

**Comment by ruflin** [comment]
This should not be merged before discussion in https://github.com/elastic/beats/pull/9460 is resolved.

**Comment by ruflin** [comment]
PR was changed to use `log.source.address`

File: filebeat/_meta/fields.common.yml [hunk 1]
```diff
@@ -116,6 +116,16 @@
       description: >
         This field contains the flags of the event.
 
+    - name: log.source.address
+      type: keyword
+      description: >
+        Log source address.
```
[the remark this answers is not in the document]
Comment (webmat) [reply filebeat/_meta/fields.common.yml]: Gotcha

File: x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
         "http.request.method": "GET", 
         "http.response.status_code": "200", 
         "input.type": "log", 
+        "log.file.path": "/Users/ruflin/Dev/gopath/src/github.com/elastic/beats/x-pack/filebeat/module/suricata/eve/test/eve-alerts.log",
```
Comment (webmat) [inline x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json]: These paths are still present in x-pack

Comment (ruflin) [reply x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json]: will update these files too and push again.

File: filebeat/module/icinga/startup/test/test.log-expected.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [
     {
-        "@timestamp": "2018-12-11T08:08:07.894Z", 
+        "@timestamp": "2018-12-12T11:22:05.182Z",
```
Comment (webmat) [inline filebeat/module/icinga/startup/test/test.log-expected.json]: Don't think it's a problem, but why are these timestamps still changing?

Comment (ruflin) [reply filebeat/module/icinga/startup/test/test.log-expected.json]: I fixed that with https://github.com/elastic/beats/pull/9506. It's not an issue for CI.

**Pull request merged**